templates/addons/calico.yaml

```diff
@@ -629,7 +629,7 @@ spec:
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
             # Disable IPv6 on Kubernetes.
-            - name: FELIX_IPV6SUPPORT
+            - name: FELIX_IPv6SUPPORT
```

Comment:
@jsturtevant why did you change this? In https://docs.projectcalico.org/reference/felix/configuration it's documented as `FELIX_IPV6SUPPORT`

Reply:
this is not the ipv6 calico file which does have it: https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/f2d0778e465c3185be58aa23e3d3c2ad6f686dcd#diff-78a399e736d7774d30b337556c866618R636-R637 

this looks like a find and replace error that lower-cased the `V` in the original calico file.  Should remove this and veriy the env variable is uppercased in both.